APCAppCore/APCAppCore/Library/Scheduler/APCScheduleDebugPrinter.m

```diff
@@ -249,6 +264,23 @@ + (NSString *) stringFromDate: (NSDate *) date
     return result;
 }
 
+- (NSString *) stringWithMillisecondsFromDate: (NSDate *) date
```

Comment:
My question was why not use class method directly? It seems this instance method is not required.